fs/report.go

```diff
@@ -224,6 +224,26 @@ func eqEntry(path string, x, y dirEntry) []failure {
 	return nil
 }
 
+func matchGlob(name, eType string, globs []string) []problem {
+	var p []problem
+
+	for _, glob := range globs {
+		ok, err := filepath.Match(glob, name)
+		if err != nil {
+			p = append(p, errProblem("failed to match glob pattern", err))
+		}
+		if ok {
```

Comment:
Now that globs is a `map[string]*filePath`:

```go
for glob, expectedFile := range filepathGlobs {
    ok, err := filepath.Match(glob, name)
    ...
    if ok {
       return eqEntry(name, expectedFile.file, yEntry)
    }
}
```

I guess this will also have to store all the matches filename as well for the `anyFile` logic.